MRI_Data/MF0302-PRE/config.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Can you please remove the config.xml and MF0302-PRE folder from the repository? 

Typically you only want to put relevant code in the repository.